src/CommandPaletteCommand.js

```diff
@@ -8,13 +8,15 @@ type CommandPaletteCommandProps = {
     commandName: string,
     intl: any,
     selectedCommandName: ?string,
+    audioManager: Object,
```

Comment:
I remember I set audioManager's type to be Object, but According to flow documentation, "NOTE: For new code, prefer any or { [key: string]: any}. Object is an alias to any and will be deprecated and removed in a future version of Flow.". We should change its type to any or import AudioManger to CommandPaletteCommand then use it as type.

Reply:
Thanks Daniel! I've updated `CommandPaletteCommand` to use the `AudioManager` type.